pkg/kubenest/init.go

```diff
@@ -35,6 +35,7 @@ type initData struct {
 	virtualClusterDataDir string
 	privateRegistry       string
 	externalIP            string
+	externalIps           []string
```

Comment:
add `externalDonmains` ?

Reply:
yes, externalIp support add domain names, ipv4 and ipv6